app/src/main/java/com/example/benjaminsalamon/msoplaner/Day.java

```diff
@@ -0,0 +1,64 @@
+package com.example.benjaminsalamon.msoplaner;
+
+class Day {
```

Comment:
background classes are package local. Some methods and variables have wrong scopes. Applies to Day, Lesson, List, Logic, Stack and Subject.